src/garage/torch/q_functions/continuous_mlp_q_function.py

```diff
@@ -0,0 +1,40 @@
+"""This modules creates a continuous MLP Q fcuntion network."""
+
+
+class ContinuousMLPQFunction:
+    """
+    This class implements a q value network.
+
+    It predicts the q value for all actions based on the input state. It uses
```

Comment:
Q-value

Reply:
Hmm all actions? What if the action space is continuous instead of discrete?